cmd/cnab-to-oci/fixup.go

```diff
@@ -72,3 +84,16 @@ func runFixup(opts fixupOptions) error {
 	}
 	return ioutil.WriteFile(opts.output, bundleJSON, 0644)
 }
+
+func displayEvent(out io.Writer, ev remotes.FixupEvent) {
+	switch ev.EventType {
+	case remotes.FixupEventTypeCopyImageStart:
+		fmt.Fprintf(out, "Starting handling image %s... ", ev.SourceImage)
+	case remotes.FixupEventTypeCopyImageEnd:
+		if ev.Error != nil {
+			fmt.Fprintf(out, "Failed: %s\n", ev.Error)
+		} else {
+			fmt.Fprint(out, "Done!\n")
```

Comment:
Should these two include `ev.SourceImage` too?

Reply:
For now images event are not overlapping, but you're right, if we do that at some poit, it might display pure garbage